app/assets/javascripts/active_admin.js

```diff
@@ -88,18 +88,37 @@ const MAX_DESCRIPTION_LENGTH = 140;
 
   function getPageDescriptionCharacterCountOnPageLoad() {
       let descriptionInput = $('#page_description');
-
       let characterCounterEle = $('.page-description-character-counter');
-      $(characterCounterEle).css('color', CHARACTER_COUNTER_VALID_COLOR);
-
-      let descriptionCurrentLength = descriptionInput.val().length;
-      let characterCounterText = '(' + descriptionCurrentLength + '/' + MAX_DESCRIPTION_LENGTH + ' characters)';
+      if (descriptionInput.length > 0 && characterCounterEle.length > 0) {
```

Comment:
⚠️ Was seeing errors in the console because it was trying to count the characters for pages that didn't have this class so I added a check to this